### PR TITLE
Set profile image

### PR DESCRIPTION
**Pull request by Simon-Laux**
also fix the instant closing bug of the color picker.

The styling could be improved before releasing this in the wild. The plan is to use symbols instead of the text and style the buttons similar to the other buttons.

### CODE REVIEW

File: _locales/_untranslated_en.json [hunk 1]
```diff
@@ -34,5 +34,17 @@
   },
   "switch_account_desktop":{
     "message": "Switch account"
+  },
+  "select_profile_image_desktop": {
```
Comment (OzancanKaratas) [inline _locales/_untranslated_en.json]: Please remove “desktop” prefix. Other clients will use this.

File: _locales/_untranslated_en.json [hunk 2]
```diff
@@ -34,5 +34,17 @@
...
+    "message": "Select your new Profile image"
```
Comment (OzancanKaratas) [inline _locales/_untranslated_en.json]: Fix typo. P must be small.

File: _locales/_untranslated_en.json (hunk 1 above)
Comment (Simon-Laux) [reply _locales/_untranslated_en.json]: this is desktop specific, its the title of the file picker window.

File: _locales/_untranslated_en.json [hunk 3]
```diff
@@ -34,5 +34,17 @@
   },
   "switch_account_desktop":{
     "message": "Switch account"
+  },
+  "select_profile_image_desktop": {
+    "message": "Select your new profile image"
+  },
+  "a11y_profile_image_label":{
```
Comment (r10s) [inline _locales/_untranslated_en.json]: do we really need a a11y string for every button and control? maybe, we could just use the normal "Your profile image" or just "Profile image" string here.

File: _locales/_untranslated_en.json (hunk 1 above)
Comment (r10s) [reply _locales/_untranslated_en.json]: hm, in general i agree with @OzancanKaratas , we should create os-specific strings only if it happens that we really need them. for most other hints etc. we do not have specific stuff.
This "Select your new profile image" might be useful on other os as well.
however, we can streamline this when adding it to androids strings.xml, this should not block merging.

File: _locales/_untranslated_en.json (hunk 3 above)
Comment (Simon-Laux) [reply _locales/_untranslated_en.json]: every button that will have no text, plan is to use a symbol instead of the text.

**Comment by Jikstra** [comment]
Styling is a bit ugly but first make it work, then make it look good. BTW, where did the border of the dialog header go? Did you & evelyn remove it at some point?

File: _locales/_untranslated_en.json (hunk 3 above)
Comment (r10s) [reply _locales/_untranslated_en.json]: k, but the having a simple `your_profile_image` would do the job - if can be used as a input field label as well as for a11y. i would use explicit a11y strings only if they are really different from some labels, same as for _desktop, _android etc. suffixes.

**Comment by r10s** [comment]
> Approving besides the a11y thingies, not sure what to do about them.

as the strings are currently only in untranslated.* it should be no problem, we can figure things out these days when merging the strings.

**Pull request merged**